### DISCUSSION TITLE
Organizational Admin Password Reset

### DISCUSSION THREAD

**Discussion by CFox243** [dani-garcia/vaultwarden#3358] [Q&A] [answered]
I see in Feature Requests that Add Organizational Admin Password Reset support is done (https://github.com/dani-garcia/vaultwarden/pull/1326). However, I don't believe it has been compiled into a docker image yet, correct?

**Comment by stefan0xC** [comment, marked as answer]
It has been implemented in #3116 and it is already available in the `testing` image (which is based of the `main` branch).
